### PR TITLE
Move subscription module to accounts

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
- [x] Move to "Accounts"
- [x] Link in `module/Accounts`
- [x] Move documentation in Accounts
- [x] Status = 'Active' / 'Stopped'
- [x] Custom button for 'Stop', 'Resume'
- [x] Remove Cancel Permission

Fixed https://github.com/frappe/erpnext/issues/10759, https://github.com/frappe/erpnext/issues/10760

### CODE REVIEW

File: erpnext/templates/emails/recurring_document_failed.html [hunk 1]
```diff
@@ -2,11 +2,10 @@
 
 <p>An error occured while creating recurring {{ type }} <b>{{ name }}</b> for <b>{{ party }}</b>.</p>
```
Comment (mnatalia) [inline erpnext/templates/emails/recurring_document_failed.html]: just a question, don't you think it should be possible to translate this message?

Comment (rohitwaghchaure) [reply erpnext/templates/emails/recurring_document_failed.html]: Thanks for your review, will fix it!

Comment (mnatalia) [reply erpnext/templates/emails/recurring_document_failed.html]: sorry again but I think all sentences in the recurring_document_failed.html should have possibility to be translated

Comment (rohitwaghchaure) [reply erpnext/templates/emails/recurring_document_failed.html]: Can you please send PR after merging of this PR?

Comment (nataliamm) [reply erpnext/templates/emails/recurring_document_failed.html]: ok

Comment (mnatalia) [reply erpnext/templates/emails/recurring_document_failed.html]: ok

**Pull request merged**